NumLib/DOF/DOFTableUtil.cpp

```diff
@@ -36,7 +36,12 @@ double norm(GlobalVector const& x, unsigned const global_component,
         }
     }
 
-    // TODO for PETSc some global accumulation is necessary.
+#ifdef USE_PETSC
```

Comment:
Just want to mention:
1. The local norms cannot simply be summed if one uses l-infinity norm
2. ghost nodes could be excluded in the loop above, maybe. Is there a performance impact if one only fetches data local to some rank?